README.md

```diff
@@ -300,11 +324,27 @@ For that, the Parser supports the ability to stringify a parsed AsyncAPI documen
 To parse a stringified document into an AsyncAPIDocument instance, you must use the `unstringify` function (also exposed by package). It isn't compatible with the native `JSON.parse()` method. It replaces the given references pointed by the [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901) path, with an `$ref:` prefix to the original objects.
 
 A few advantages of this solution:
+
 - The string remains as small as possible due to the use of [JSON Pointers](https://datatracker.ietf.org/doc/html/rfc6901).
 - All references (also circular) are preserved.
 
 Check [example](#example-with-stringify-and-unstringify-parsed-documentstringify).
 
+## Convert to the old API
+
+Version `2.0.0` of package introduced a lot of breaking changes, including changing the API of the returned parsed document (parser uses [Intent API](https://github.com/asyncapi/parser-api)). Due to the fact that a large part of the AsyncAPI tooling ecosystem uses a Parser with the old API and rewriting the tool for the new one can be time-consuming and difficult, the package exposes the `migrateToOldAPI()` function to convert new API to old one:
+
+```js
+import { Parser, migrateToOldAPI } from '@asyncapi/parser';
+
+const parser = new Parser();
+const { document } = parser.parse(...);
+const oldAsyncAPIDocument = migrateToOldAPI(document);
```

Comment:
I do have a question: Would it make sense to, instead of having an isolated function here that requires an already parsed document, just create another parse function? E.g. `parser.parseWithOldAPI(...)`. It feels more user-friendly to me. Are there any technical cons?

Reply:
Maybe it is more user friendly, but this method will add all the old API to the final bundle for browser applications. With a separate method we can import the old api whenever we need it and not when it is optional (we will broke treeshaking support)

Reply:
Fair enough and well explained 👍 👍